libs/editor/WordPressEditor/src/main/res/values/colors.xml

```diff
@@ -1,18 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
+
 <resources>
+
+    <color name="format_bar_background">#f9fbfc</color>
+    <color name="format_bar_button_highlighted_color">@color/wp_blue</color>
+    <color name="format_bar_button_normal_color">@color/wp_grey_lighten_10</color>
```

Comment:
I cant find any usage of this two resources (format_bar_button_xx). Should we remove them?